shairport.c

```diff
@@ -2102,6 +2108,18 @@ int main(int argc, char **argv) {
   // parse arguments into config -- needed to locate pid_dir
   int audio_arg = parse_options(argc, argv);
 
+#ifdef COMPILE_FOR_OPENBSD
+  /* Drop "proc exec" unless external commands are to be run. */
+  if ((config.cmd_active_start == NULL) &&
+      (config.cmd_active_stop == NULL) &&
+      (config.cmd_start == NULL) &&
+      (config.cmd_stop == NULL) &&
+      (config.cmd_set_volume == NULL)) {
```

Comment:
`config.unfixable` aka. `run_this_if_an_unfixable_error_is_detected` is ALSA only, which OpenBSD does not support, thus ignored here.

Did I miss any other command?
Easy to glance over `-h` and only see `-B/--cmd-start` and `-E/--cmd-stop` as the others are only documented in the configuration file.